lib/src/services/user_service.dart

```diff
@@ -16,12 +15,36 @@ class UserService {
     return (await all()).firstWhere((user) => user.uid == uid);
   }
 
+  Future<Map<String, UserModel>> asMap() async {
+    Map<String, UserModel> uidToUser = {};
+    List<UserModel> users = await all();
+
+    for (UserModel user in users) {
+      uidToUser.putIfAbsent(user.uid, () => user);
+    }
+
+    return uidToUser;
+  }
+
   Future<List<UserModel>> all() async {
     String data = await rootBundle.loadString('assets/mock/data/users.json');
     final List jsonData = await jsonDecode(data);
-    final List<UserModel> usersList =
+    List<UserModel> usersList =
         jsonData.map((user) => UserModel.fromJson(user)).toList();
 
+    Random random = Random();
+    usersList = usersList
+        .map(
+          (user) => user.copyWith(
+            imageUrl: 'assets/mock/pictures/profile-' +
```

Comment:
in dart gibts viel schönere String templates ganz ohne extra Zeichen wie in JS

'${myVar.someProperty} ist doch $easy' :)